renovate.json

```diff
@@ -27,5 +27,8 @@
     "schedule": [
       "after 22:00 and before 6:00 every weekday"
     ]
+  },
+  "github-actions": {
+    "fileMatch": ["(^\\.github\\/workflows)\\/[^/]+\\.ya?ml$"]
```

Comment:
Would this ignore .j2 workflow files?

Reply:
Oof, I thought I changed this to include `template/`.

> Would this ignore .j2 workflow files?

Yes, because it's not really practical for Renovate to modify Jinja2 templates.